README.md

```diff
@@ -0,0 +1,14 @@
+### Starting up this project
+
+1. Install npm packages 
+
+```
+npm install
+```
+
+2. Create file `.env` in root of project folder
+3. Insert in this file key from unsplash.com else you wont see images in cards
+
```

Comment:
good catch! 
Extend documentation a little bit, provide also a required node version. Help other developers to save time on it.